### PR TITLE
Add testing for tf ops initialization

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: qa/common/gen_qa_tf_parameters.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    tf.reset_default_graph()
+    input0 = tf.placeholder(tf.int32, [
+        1,
+    ], "INPUT")
+    variable = tf.get_variable("VARIABLE", [
+        1,
+    ], initializer=tf.zeros_initializer(), dtype=tf.int32)
+    tf.add(variable, input0, name="OUTPUT")
+    print(tf.global_variables_initializer())
```
[the remark this answers is not in the document]
Comment (Tabrizian) [reply qa/common/gen_qa_tf_parameters.py:49]: It is not needed. I just wanted to find out the op name. I initially though it is the variable name but looks like there was a separate operation for initialization. I'll remove this line.

File: qa/common/gen_qa_tf_parameters.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```
Comment (tanmayv25) [inline qa/common/gen_qa_tf_parameters.py:1]: Need to update the years. Here and everywhere.

**Review by tanmayv25 — CHANGES_REQUESTED** [review]
small change other wise looks good to me.

**Review by tanmayv25 — COMMENTED** [review]
Looks good to me. Not approving to prevent accidental merge of the PR before we move to 22.07 TF.

**Review by tanmayv25 — APPROVED** [review]

**Review by tanmayv25 — APPROVED** [review]

File: qa/L0_tf_tag_sigdef/test.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+# Copyright 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
Comment (jbkyang-nvi) [inline qa/L0_tf_tag_sigdef/test.sh:2]: nit: 2021-2022?

Comment (tanmayv25) [reply qa/L0_tf_tag_sigdef/test.sh:2]:
```suggestion
# Copyright 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**